packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.test.tsx

```diff
@@ -272,58 +248,6 @@ describe('QueryWrapper', () => {
     })
   })
 
-  describe('locked facet', () => {
```

Comment:
Removing data based on a locked facet now happens somewhere else, because `QueryWrapper` doesn't store table query data, so remove the tests here